docs/useCases.md

```diff
@@ -1363,3 +1365,23 @@ getZipDownloadLimit.execute().then((downloadLimit: number) => {
 ```
 
 _See [use case](../src/info/domain/useCases/GetZipDownloadLimit.ts) implementation_.
+
+## Contact
+
+#### Send Feedback to Object Contacts
+
+Returns a [Contact](../src/contactInfo/domain/models/Contact.ts) object, which contains contact return information, showing toEmail, fromEmail, subject, body.
+
+##### Example call:
+
+```typescript
+import { submitContactInfo } from '@iqss/dataverse-client-javascript'
+
+/* ... */
+
+submitContactInfo.execute((contact: ContactDTO)).then( => {
+/* ... */
+})
+
+/* ... */
+```
```

Comment:
Typically, we add the phrase '_See [use case](../src/datasets/domain/...)_ definition.' right below the code. Please take a look at other examples. Additionally, it would be helpful to provide a high-level description of the DTO or at least mention the type of identifier (targetId) it handles (numeric database identifier vs persistent identifiers). I assume it uses the latter, since it is numeric and is the common one for all DV objects, but it would be good to clarify.

On the other hand, the syntax in the block is incorrect. You need to first create an instance of the test DTO and then pass it as a parameter. (contact: ContactDTO) is how the parameter is defined in the method, not how it is invoked.